File: 02-eda-1/boxplot.r
df <- read.csv("movies.csv")
df <- na.omit(df)
# Convert Gross column from string to float
df$Gross <- as.numeric(gsub("[$M]", "", df$Gross))

boxplot(df$Gross, df$RunTime,
        main = "Boxplot of Gross and RunTime",
        names = c("Gross", "RunTime"),
        ylab = "Values")

File: 02-eda-1/boxplot.py
import pandas as pd
import matplotlib.pyplot as plt

df = pd.read_csv("movies.csv")
df = df.dropna(subset=["Gross", "RunTime"])
# Convert Gross column from string to float
df["Gross"] = (df["Gross"].str.replace("$", "")).str.replace("M", "").astype(float)

df[["Gross", "RunTime"]].plot(kind="box", title="boxplot")
plt.show()
